samples/cellular/modem_shell/src/location/location_shell.c

```diff
@@ -282,9 +385,20 @@ void location_ctrl_event_handler(const struct location_event_data *event_data)
 		location_print_data_details(event_data->method, &event_data->location.details);
 #endif
 		if (gnss_location_to_cloud) {
+			k_work_init(&gnss_location_work_data.work, location_to_cloud_worker);
```

Comment:
A work item must not be initialized again unless it's idle. I'm not sure if we can guarantee it here, so I think it would be better to perform the initialization only once somewhere else.

Reply:
This is already in `location_ctrl_init()`